modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} (OCP) update process.
+
+. The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource, which may be managed through the web console or the CLI.
+
+. The Cluster Version Operator (CVO) discovers that the `desiredUpdate` in the ClusterVersion resource differs from the current cluster version.
+Using graph data from the OpenShift Update Service, the CVO resolves the desired cluster version to a pullspec to the release image.
+
+. The CVO validates the integrity and authenticity of the release image.
+Red Hat publishes cryptographically-signed statements about published release images at predefined locations, using image SHA digests as unique and immutable release image identifiers.
+The CVO utilizes a list of built-in public keys to validate the presence and signatures of the statement matching the checked release image.
+
+. The CVO creates a job named `version-$version-$hash` in the `openshift-cluster-version` namespace.
+This job uses containers that are executing the release image, so the cluster first downloads the image through the container runtime.
```

Comment:
You can probably s/cluster first/cluster 
cluster first sounds like an adjective (cluster-first actions, or something)